implementations/rust/ockam/ockam_api/src/kafka/portal_worker.rs

```diff
@@ -60,7 +60,7 @@ pub(crate) struct KafkaPortalWorker {
 
 #[ockam::worker]
 impl Worker for KafkaPortalWorker {
-    type Message = PortalMessage;
+    type Message = NeutralMessage;
```

Comment:
I think we could move the parameter declarion to the `message.into_body()` call and by doing so, we could also allow messages that borrows data and avoid the manual encoding/decoding calls.
On the other hand It's simpler for the avarage user to think a worker with a fixed type.
Maybe we could move the message type from the worker to the incoming addresses instead, but i don't have a clear picture on how it would look like.